test/export/ValueSetExporter.test.ts

```diff
@@ -0,0 +1,229 @@
+    input = new FSHTank([doc], { name: 'test', version: '0.0.1', canonical: 'http://example.com' });
+    exporter = new ValueSetExporter(defs, input);
+  });
+
+  it('should output empty results with empty input', () => {
+    const exported = exporter.export();
+    expect(exported).toEqual([]);
+  });
+
+  it('should export a single value set', () => {
+    const valueSet = new FshValueSet('BreakfastVS');
+    doc.valueSets.set(valueSet.name, valueSet);
+    const exported = exporter.export();
+    expect(exported.length).toBe(1);
+    expect(exported[0].name).toBe('BreakfastVS');
+    expect(exported[0].id).toBe('BreakfastVS');
+    expect(exported[0].title).toBeUndefined();
+    expect(exported[0].description).toBeUndefined();
+    expect(exported[0].url).toBe('http://example.com/ValueSet/BreakfastVS');
+  });
+
+  it('should export multiple value sets', () => {
+    const breakfast = new FshValueSet('BreakfastVS');
+    const lunch = new FshValueSet('LunchVS');
+    doc.valueSets.set(breakfast.name, breakfast);
+    doc.valueSets.set(lunch.name, lunch);
+    const exported = exporter.export();
+    expect(exported.length).toBe(2);
+  });
+
+  it('should export a value set with additional metadata', () => {
+    const valueSet = new FshValueSet('BreakfastVS');
+    valueSet.title = 'Breakfast Values';
+    valueSet.description = 'A value set for breakfast items';
+    doc.valueSets.set(valueSet.name, valueSet);
+    const exported = exporter.export();
+    expect(exported.length).toBe(1);
+    expect(exported[0].name).toBe('BreakfastVS');
+    expect(exported[0].id).toBe('BreakfastVS');
+    expect(exported[0].title).toBe('Breakfast Values');
+    expect(exported[0].description).toBe('A value set for breakfast items');
+    expect(exported[0].url).toBe('http://example.com/ValueSet/BreakfastVS');
+  });
+
+  it('should export each value set once, even if export is called more than once', () => {
+    const breakfast = new FshValueSet('BreakfastVS');
+    doc.valueSets.set(breakfast.name, breakfast);
+    let exported = exporter.export();
+    expect(exported.length).toBe(1);
+    exported = exporter.export();
+    expect(exported.length).toBe(1);
+  });
+
+  it('should export a value set that includes a component from a system', () => {
+    const valueSet = new FshValueSet('DinnerVS');
+    const component = new ValueSetConceptComponent(true);
+    component.from = { system: 'FOOD' };
+    valueSet.components.push(component);
+    doc.valueSets.set(valueSet.name, valueSet);
+    const exported = exporter.export();
+    expect(exported.length).toBe(1);
+    expect(exported[0].compose.include.length).toBe(1);
+    expect(exported[0].compose.include[0].system).toBe('FOOD');
+  });
+
+  it('should export a value set that includes a component from a value set', () => {
+    const valueSet = new FshValueSet('DinnerVS');
+    const component = new ValueSetConceptComponent(true);
+    component.from = { valueSets: ['HotFoodVS', 'ColdFoodVS'] };
```

Comment:
Same as above (and I won't mention it for anything else) -- technically a string like `HotFoodVS` is not a valid VS identifier.  It would be a more realistic test if it were a URL.